.github/workflows/ci.yml

```diff
@@ -118,49 +41,3 @@ jobs:
 
       - name: Check fixtures
         run: ./run-tests.sh --check-fixtures
```

Comment:
Note that we also spoke about strict schema checking both on the DB side and the OpenSearch side. Have you created the checker? Can you add it to the `run-tests.sh` and the CI?  We should check any troubles early.

Reply:
It would be better to add that new functionality later on. This PR is already big enough.